config/sem_dedup_config.yaml

```diff
@@ -20,14 +20,8 @@ random_state: 1234
 sim_metric: "cosine"
 which_to_keep: "hard"
 batched_cosine_similarity: 1024
-sort_clusters: true
-kmeans_with_cos_dist: false
 clustering_input_partition_size: "2gb"
 
 # Extract dedup configuration
-eps_thresholds:
-  - 0.01
-  - 0.001
-
 # Which threshold to use for extracting deduped data
 eps_to_extract: 0.01
```

Comment:
Doesn't have to be in this PR, but it might be clearer to users to rename this to something else.

Reply:
Agreed. Want to make a few follow up changes
1. Output a removal list instead of `docs_to_keep` and
2. Ability to perform removal too.

Maybe in that PR we can consider making this change